plugins/woocommerce/client/admin/client/payments/payment-recommendations.tsx

```diff
@@ -262,10 +260,32 @@ const PaymentRecommendations: React.FC = () => {
 			</CardHeader>
 			<List items={ pluginsList } />
 			<CardFooter>
-				<Button href={ SEE_MORE_LINK } target="_blank" isTertiary>
-					{ __( 'Discover other payment providers', 'woocommerce' ) }
-					<ExternalIcon size={ 18 } />
-				</Button>
+				<Text>
+					{ interpolateComponents( {
+						mixedString: __(
+							'Visit the {{sbLink}}Official WooCommerce Marketplace{{/sbLink}} to find additional payment providers.',
+							'woocommerce'
+						),
+						components: {
+							sbLink: (
+								<Link
+									onClick={ () => {
+										recordEvent(
+											'settings_payment_recommendations_visit_marketplace_click',
+											{}
+										);
+										window.location.href = getAdminLink(
+											'admin.php?page=wc-admin&tab=extensions&path=/extensions&category=payment-gateways'
+										);
+										return false;
+									} }
+									href=""
```

Comment:
I'm wondering if it's possible to combine `onClick` with `href` like this?

```suggestion
								<Link
									onClick={ () => {
										recordEvent(
											'settings_payment_recommendations_visit_marketplace_click',
											{}
										);
										return true;
									} }
									href={ getAdminLink( 'admin.php?page=wc-admin&tab=extensions&path=/extensions&category=payment-gateways' }
```

Reply:
This doesn't work: if the `Link` type is `wc-admin` and the `href` is a wc-admin route, it uses the `wcAdminLinkHandler` defined in that component which, among other things, prevents default navigation to the href URL.

We could change the link type to `external`, which will let the href work, but given this is, in fact, a wc-admin link that might have unforeseen side-effects, or at the very least cause confusion for the next person to edit this code.

We could also use a regular `<a>` tag instead of `<Link>`, but I tried to follow a consistent approach with other links I saw in the different task list steps.

Reply:
Feel free to dismiss, not a blocker.

(FWIW I achieved something similar using a `Button` [here](https://github.com/woocommerce/woocommerce/blob/d4a65ef5df39906d422ec4de94cf465f530357ca/plugins/woocommerce/client/admin/client/homescreen/layout.js#L192-L202))

Reply:
It works for the `Button` component, because it doesn't have a wc-admin specific click handler that's interfering with the default behaviour.